University/Nikita/Person.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace University.Nikita
+{
+    class Person : IReadOnlyPerson
+    {
+        public string FullName { get; set; }
```

Comment:
why set here is not private?
why FullName isn't formed from first and last names?